### ISSUE TITLE
Corneal iron line

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10875]
Preferred term label:
Corneal iron line

Synonyms
Iron deposition in the cornea, Corneal iron deposits

Definition (free text, please give PubMed ID)
Linear or arcuate deposition of iron within the corneal epithelium or stroma, typically associated with conditions such as keratoconus, corneal degeneration, or postsurgical corneal changes.

Parent term (use hpo.jax.org/app)
Abnormality of the corneal epithelium (11495)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Keratoconus (OMIM: 148300)

Your nano-attribution (ORCID)
0000-0001-9042-0508
